src/config.py

```diff
@@ -0,0 +1,40 @@
+
+import os
+
```

Comment:
Is there a way to make settings differentiated between user modifiable and non-modifiable (not needed/useful to modify)?

Reply:
There's no way to do that in Python other than making every single value a getter/setter pair, which seems excessive. That's just how things are in Python.